src/components/ControlGroup/EnumControl.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import { describe, expect, it } from "bun:test"
+import { render, screen, fireEvent } from "@testing-library/react"
+import { EnumControl } from "./EnumControl";
+import { jest } from "bun:test";
+
+describe("EnumControl", () => {
+    it('Should render discription.', () => {
+        const legendDiscription = `legendDiscription`
+        render(<EnumControl description={legendDiscription} options={[]} setPosition={() => {}} />)
+
+        const discription = screen.getByText('legendDiscription')
+        expect(discription).toBeInTheDocument();
+    });
+    it("Should render given number of options as lables.", async () => {
+        const options = ['top', 'bottom', 'right', 'left'];
+        render(<EnumControl options={[...options]}  setPosition={() => {}}/>)
```

Comment:
No need to spread options here; `options={options}` is fine.
`setPosition={Object}` purely just because we use `Object` as a no-op'ish function in alot of other places